docs/references/astro/migrating-from-astro-community-sdk.mdx

```diff
@@ -0,0 +1,184 @@
+---
+
+# Migrating from the Astro community SDK
+
+In July 2024, we introduced official support for Astro. This migration guide covers converting from the [`astro-clerk-auth`](https://github.com/panteliselef/astro-with-clerk-auth) community SDK to the official SDK. It covers the breaking changes that were introduced and provides examples on how to resolve them.
+
+
+**Overview:**
+- Install `@clerk/astro`, uninstall `astro-clerk-auth`
+- Update import paths
+- Replace deprecated APIs
+
+## Installation
+
+Uninstall the community SDK and install the new offical SDK for Astro.
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash filename="terminal"
+    npm uninstall astro-clerk-auth
+    npm install @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    yarn remove astro-clerk-auth
+    yarn add @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    pnpm remove astro-clerk-auth
+    pnpm add @clerk/astro
+    ```
+</CodeBlockTabs>
+
+## Breaking Changes
+
+### The integration
+
+When using the integration from `@clerk/astro` we are now hotloading clerk-js by default.
+Previously you could choose between two strategies when using the clerk intergration. Now only one is offered:
+
+**Hotload** 
+
+```diff
+import { defineConfig } from "astro/config";
+import node from "@astrojs/node";
+- import clerk from "astro-clerk-auth/hotload";
++ import clerk from "@clerk/astro";
+
+- import clerk from "astro-clerk-auth";
++ import clerk from "@clerk/astro";
+
+- import clerk from "astro-clerk-auth/integration";
++ import clerk from "@clerk/astro";
+
+- import clerk from "astro-clerk-auth/integration/hotload";
++ import clerk from "@clerk/astro";
+
+export default defineConfig({
+    integrations: [clerk()],
+    output: "server",
+    adapter: node({ mode: "standalone" }),
+});
+```
+
+**Bundled** \
+We have completely removed the option for an integration that uses the bundled clerk-js variant
+
+### Prefix for environment variable
+
+We changed the expected required prefix for environment variables available in client-side code. This was to align with [best practises](https://docs.astro.build/en/guides/environment-variables/) by Astro.
+
+
+```diff
+
+- PUBLIC_ASTRO_APP_CLERK_PUBLISHABLE_KEY=
++ PUBLIC_CLERK_PUBLISHABLE_KEY=
+
+- PUBLIC_ASTRO_APP_CLERK_SIGN_IN_URL=
++ PUBLIC_CLERK_SIGN_IN_URL=
+
+- PUBLIC_ASTRO_APP_*=
++ PUBLIC_*=
+
+```
+
+
+### Changes to Astro components
+
+```diff
+
+- import { UserProfile } from "@clerk/astro/components/interactive"
++ import { UserProfile } from "@clerk/astro/components"
+
+- import { Protect } from "@clerk/astro/components/control"
++ import { Protect } from "@clerk/astro/components"
+
+- import { SignInButton } from "@clerk/astro/components/unstyled"
++ import { SignInButton } from "@clerk/astro/components"
+
+```
+
+### Changes related to stores
+
+Submodule `/stores` was replaced with `/client`. For the full list of the stores see [here](TODO: add reference)
```

Comment:
```suggestion
Submodule `/stores` was replaced with `/client`. For the full list of the stores see [here](docs/references/astro/overview#client-side-helpers)
```

I opted to link to the Overview page and the section for stores since that lists all. Please adjust as desired. There is no Overview just for the stores to link to.

Reply:
I think this is acceptable, we do this for nextjs if i recall correctly